### PR TITLE
Update CHANGELOG to mention that @autoclosure forwarding is disabled …

### PR DESCRIPTION
**Pull request by xedin**
…in Swift >= 5

### CODE REVIEW

**Comment by xedin** [comment]
@AnnaZaks Does this explanation make sense to you?

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ CHANGELOG
 
 Swift 5.0
 ---------
+* [SR-9494][]:
+
+  Implicit `@autoclosure` function type forwarding has been disabled,
```
Comment (slavapestov) [inline CHANGELOG.md]: Please add a note that this only happens in -swift-version 5 mode

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,22 @@ CHANGELOG
 
 Swift 5.0
 ---------
+* [SR-9494][]:
+
+  Starting from `-swift-version 5` implicit `@autoclosure` function type
```
Comment (xedin) [inline CHANGELOG.md]: Does this look better, @slavapestov ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,22 @@ CHANGELOG
...
+  forwarding has been disabled and new diagnostic has been added suggesting
```
Comment (slavapestov) [inline CHANGELOG.md]: comma: "Starting from -swift-version
comma: "has been disabled,"
missing 'a': "and a new diagnostic"

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,6 +23,22 @@ CHANGELOG
...
+  to add `()` to call function type in such case. Call itself would be
```
Comment (slavapestov) [inline CHANGELOG.md]: "to call the function value"
"The call itself"

File: CHANGELOG.md [hunk 5]
```diff
@@ -23,6 +23,22 @@ CHANGELOG
...
+  wrapped in implicit closure and passed to a corresponding parameter.
```
Comment (slavapestov) [inline CHANGELOG.md]: "in an implicit"
"passed to the"

**Review by slavapestov — COMMENTED** [review]
Some minor edits

**Comment by xedin** [comment]
@slavapestov All fixed, thank you and sorry for completely illiterate.

**Comment by benrimmington** [comment]
@xedin You'll need to add a URL to the end of this CHANGELOG.md file.

[SR-9494](https://bugs.swift.org/browse/SR-9494) is probably a duplicate of [SR-5719](https://bugs.swift.org/browse/SR-5719) (also discussed on [forums](https://forums.swift.org/t/how-should-we-fix-sr-5719-for-swift-5/8777)).

**Comment by xedin** [comment]
@benrimmington Thank you! I have so many JIRAs by now that I just overlooked that one... I've replaced SR-9494 with SR-5719 (duplicated it in JIRA), and added a link.

**Comment by xedin** [comment]
@swift-ci please smoke test

**Comment by xedin** [comment]
Going to merge a bit later today unless anybody objects.

**Pull request merged**